test/components/FeatureFlagRenderer.test.js

```diff
@@ -61,7 +63,7 @@ describe("components/FeatureFlagRenderer", () => {
           />
         );
         wrapper.setState({ flagValue: true });
-        expect(wrapper.text()).toEqual(renderFeatureCallback());
+        expect(wrapper.text()).toEqual(renderFeatureCallback().props.children);
```

Comment:
Now that I am looking at this again, maybe this can be simplified to:
`expect(wrapper).toEqual(renderFeatureCallback());`